pyproject.toml

```diff
@@ -1,3 +1,29 @@
+[tool.poetry]
+name = "oemof-B3"
+version = "0.0.0"
+description = "Windflächenrechner - Agora Energiewende"
+authors = [
+  "Jann Launer <jann.launer@rl-institut.de>",
+  "Sabine Haas <sabine.haas@rl-institut.de>",
+  "Marie-Claire Gering <marie-claire.gering@rl-institut.de>",
+  "Hendrik Huyskens <hendrik.huyskens@rl-institut.de>",
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
```

Comment:
Python 3.7 still works - do you suggest to drop the support?

Reply:
no